### PR TITLE
Add Volunteer and news page

### PR DESCRIPTION
**Pull request by Mariatta**
- Add the Volunteer page
- Add the News page
- Add the RSS link

### CODE REVIEW

**Comment by Cheukting** [comment]
The news page needs some work on the CSS, do you want to merge this one and I open a new one to work on it? @Mariatta

**Comment by Mariatta** [comment]
I'm ok with updating the CSS later

**Comment by cmaureir** [comment]
I'd like to work later on the volunteer page, so we can hide a bit of information like we do with the code of conduct https://conference.pyladies.com/about/

File: themes/pyladies/layouts/partials/sections/home.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+{{ $author:= site.Data.author }}
```
Comment (cmaureir) [inline themes/pyladies/layouts/partials/sections/home.html:1]: is this used? @Mariatta

**Review by cmaureir — APPROVED** [review]

File: themes/pyladies/layouts/partials/sections/home.html (hunk 1 above)
Comment (Mariatta) [reply themes/pyladies/layouts/partials/sections/home.html]: Removed

**Pull request merged**